operations/setup.go

```diff
@@ -130,12 +136,49 @@ func (c *serviceConf) setup(ctx context.Context) error {
 	return nil
 }
 
-func newServiceConf(numWorkers int, localQueue bool, mongodbURI, bucket, dbName string) *serviceConf {
+type dbCreds struct {
+	DBUser string `yaml:"mdb_database_username"`
+	DBPwd  string `yaml:"mdb_database_password"`
+}
+
+func loadCredsFromYAML(filePath string) (*dbCreds, error) {
+	creds := &dbCreds{}
+
+	file, err := os.Open(filePath)
+	if err != nil {
+		grip.Error(err)
```

Comment:
nit: It seems incorrect to log the error in this case but not in the case that YAML errors. I would suggest not logging at all and instead relying on the caller.

Reply:
Good catch, this was cruft from a time of extremely indiscriminate debug logging